### PR TITLE
Removing some small/tiny dusts from recipes

### PR DESCRIPTION
**Pull request by LewisSaber**
im too stupid to use github properly reee

### CODE REVIEW

**Comment by LewisSaber** [comment]
Gotta remove small dusts from planet stone crushing, should i multiply amount of stone needed OR lower dust chance

**Comment by LewisSaber** [comment]
@Dream-Master

**Comment by Dream-Master** [comment]
will check

File: src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java [hunk 1]
```diff
@@ -1228,9 +1228,9 @@ public void run() {
...
 
-        GT_Values.RA.addCentrifugeRecipe(GT_OreDictUnificator.get(OrePrefixes.dust.get(Materials.DeepIron), 1L), GT_Values.NI, GT_Values.NF, GT_Values.NF, GT_OreDictUnificator.get(OrePrefixes.dustSmall, Materials.Iron, 1L), GT_OreDictUnificator.get(OrePrefixes.dustSmall, Materials.Iron, 1L), GT_OreDictUnificator.get(OrePrefixes.dustSmall, Materials.Iron, 1L), GT_OreDictUnificator.get(OrePrefixes.dustSmall, Materials.Trinium, 1L), GT_Values.NI, GT_Values.NI, new int[]{10000, 10000, 10000, 10000}, 200, 30720);
+        GT_Values.RA.addCentrifugeRecipe(GT_OreDictUnificator.get(OrePrefixes.dust.get(Materials.DeepIron), 4L), GT_Values.NI, GT_Values.NF, GT_Values.NF, GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Iron, 3L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Trinium, 1L), GT_Values.NI, GT_Values.NI, new int[]{10000, 10000}, 800, 30720);
```
Comment (Dream-Master) [inline src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:1233]: looks ok

File: src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java [hunk 2]
```diff
@@ -1228,9 +1228,9 @@ public void run() {
...
+        GT_Values.RA.addCentrifugeRecipe(GT_OreDictUnificator.get(OrePrefixes.dust, Materials.QuartzSand, 9L), GT_Values.NI, GT_Values.NF, GT_Values.NF, GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Stone, 1L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.CertusQuartz, 1L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.NetherQuartz, 1L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Quartzite, 1L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Cassiterite, 1L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 1L), new int[]{9000, 3000, 2000, 1500, 1000, 750}, 5400, 16);
```
Comment (Dream-Master) [inline src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:1231]: i would lower the processing time. Sure it was more expensive before but i lower it for stone dust production as well

**Review by Dream-Master — CHANGES_REQUESTED** [review]

File: src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java (hunk 2 above)
Comment (Dream-Master) [reply src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:1231]: maybe use 2400

Comment (Dream-Master) [reply src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:1231]: compared to stone dust use 108 sec = 2160

File: src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java [hunk 3]
```diff
@@ -1228,9 +1228,9 @@ public void run() {
...
 
-        GT_Values.RA.addCentrifugeRecipe(GT_OreDictUnificator.get(OrePrefixes.dust.get(Materials.DeepIron), 1L), GT_Values.NI, GT_Values.NF, GT_Values.NF, GT_OreDictUnificator.get(OrePrefixes.dustSmall, Materials.Iron, 1L), GT_OreDictUnificator.get(OrePrefixes.dustSmall, Materials.Iron, 1L), GT_OreDictUnificator.get(OrePrefixes.dustSmall, Materials.Iron, 1L), GT_OreDictUnificator.get(OrePrefixes.dustSmall, Materials.Trinium, 1L), GT_Values.NI, GT_Values.NI, new int[]{10000, 10000, 10000, 10000}, 200, 30720);
+        GT_Values.RA.addCentrifugeRecipe(GT_OreDictUnificator.get(OrePrefixes.dust.get(Materials.DeepIron), 4L), GT_Values.NI, GT_Values.NF, GT_Values.NF, GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Iron, 3L), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Trinium, 1L), GT_Values.NI, GT_Values.NI, new int[]{10000, 10000}, 800, 30720);
```
Comment (Dream-Master) [inline src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:1233]: you forgot two GT_Values.NI,  the centrifuge need 6 output slot items definied or GT_Values.NI,

**Pull request merged**